kyo-core/shared/src/test/scala/kyoTest/envsTest.scala

```diff
@@ -6,61 +6,61 @@ class envsTest extends KyoTest:
 
     "value" in {
         val v1 =
-            Envs[Int].get.map(_ + 1)
+            Envs.get[Int].map(_ + 1)
```

Comment:
This becomes much more 'readable' in my mind. 'get int' vs 'int get'

Reply:
true that